project/conf/dev/base.py

```diff
@@ -6,5 +6,5 @@
 RECAPTCHA_PUBLIC_KEY = os.getenv("RECAPTCHA_PUBLIC_KEY")
 RECAPTCHA_PRIVATE_KEY = os.getenv("RECAPTCHA_PRIVATE_KEY")
 RECAPTCHA_REQUIRED_SCORE = float(os.getenv("RECAPTCHA_REQUIRED_SCORE"))
-
 REQUIRE_SECURE_HTTP_FOR_GEOSERVER_INTROSPECTION = False
+MAP_URL = "https://example"
```

Comment:
@mattiagiupponi this can be removed. It was for a specific client, with a specific link back to their application.